.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  'extends': 'airbnb',
+  'rules': {
+    'no-console': 'off',
+    'global-require': 0,
+    'no-debugger': 'off',
+    'react/prefer-stateless-function': 'warn',
```

Comment:
Reason to make it warn instead of error?

Reply:
It's just my preferences to use 'warn'  ))